Habit-Tracker-JVR-Hannes/Habit-Tracker/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "Habit-Tracker": {
+      "commandName": "Project",
+      "workingDirectory": "C:\\Users\\Hannes\\source\\repos\\Habit-Tracker\\Habit-Tracker\\Properties\\"
```

Comment:
Be careful with hardcoding workingDirectory paths.

This path does not exist on my machine, so throws an exception on start-up.

If you are relying on this, add a useful not in your README under a Installation/Getting Started/Usage section.